frontend/views/Home.tsx

```diff
@@ -150,12 +286,117 @@ function Home(props: Props) {
             />
           );
         }}
-        //ListEmptyComponent={EmptyListMessage}
       />
     </SafeAreaView>
   );
 }
 
+/* ADMIN */
+
+/**
+ * Properties for {@link Users}
+ */
+type AdminProps = {
+  // callback called when the {@link FlatList} is refreshed
+  onRefresh: () => void;
+  // Users to render
+  users: { name: string; email: string; superUser: boolean }[];
+  navigation: any;
```

Comment:
Type can be more specific, I think it's
```suggestion
  navigation: StackNavigationProp<Routes, "Home">;
```